packages/story-editor/src/components/elementLink/frame.js

```diff
@@ -79,10 +79,6 @@ function WithLink({ element, active, children, anchorRef }) {
       </>
     ) : null;
 
-  if (!tooltipContent) {
-    return children;
-  }
```

Comment:
Any idea what this bit of code was supposed to do? Is something else not working as it should now?

Reply:
https://xwp.slack.com/archives/CCCAUAH6F/p1643723188141189?thread_ts=1643716364.949999&cid=CCCAUAH6F